policyengine_us/parameters/gov/contrib/cbo/payroll/taxable_earnings_upper_threshold.yaml

```
@@ -0,0 +1,9 @@
+description: Subject earnings greater than this amount to payroll taxes, in addition to earnings below the maximum taxable amount under current law.
+values:
+  2021-01-01: .inf
+metadata:
+  unit: USD
+  label: Earnings Upper Threshold Subject to Social Security Payroll Taxes
```

Let's also rename it `secondary_earnings_threshold.yaml`
```suggestion
  label: Social Security payroll tax secondary earnings threshold
```